sw/device/silicon_creator/rom/e2e/BUILD

```diff
@@ -52,6 +52,8 @@ MSG_MANIFEST_BAD_ENTRY_POINT = _BFV_TEMPLATE.format("014d410d")
 
 MSG_MANIFEST_BAD_CODE_REGION = _BFV_TEMPLATE.format("024d410d")
 
+MSG_ROM_BOOT_FAILED = _BFV_TEMPLATE.format("014d5209")
+
```

Comment:
This seems to be the message I get if I don't trigger a failure; is there a better way to check for success in a test that doesn't use OTTF?

Reply:
You get this message because this test uses the CRT of the ROM (doesn't initialize its CRT) so when `_ottf_start()` returns, execution returns to `rom_boot()`. Since we don't expect the second stage return back to ROM under normal conditions, we shutdown the chip with this error. I think the right thing to do here is to output a message over UART and `wfi`. Please see my comment below.